### PR TITLE
Basic project

### PR DESCRIPTION
**Pull request by sajjadrajaee**
"To-do list" is a tool that helps to organize our day. It simply lists the things that you need to do and allows you to mark them as complete. I built a simple website that allows for doing that, and you will do it using ES6 and Webpack!

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Todo List</title>
+    <script
+      src="https://kit.fontawesome.com/1a3881f3c5.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+  <div class="container">
```
Comment (digia3d) [inline src/index.html:13]: Maybe it will be better to use HTML syntax instead of using only div. For example use header with class "container".

**Review by digia3d — CHANGES_REQUESTED** [review]
Good job. The code is clean, dry, and follows JavaScript best practices. Some suggestions look in the comment below. :+1:

**Review by digia3d — APPROVED** [review]
APPROVED! :100: 
Congratulation :1st_place_medal: 
It is time to merge.

**Comment by sajjadrajaee** [comment]
Thank you for your time spent reviewing my code. you are right, it's better to use semantic tags. I will fix all changes. 👌

**Pull request merged**